src/Audit/UntrustedRoles.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Drutiny\Plugin\Drupal8\Audit;
+
+use Drutiny\Audit;
+use Drutiny\Sandbox\Sandbox;
+
+/**
+ * User #1
+ */
+class UntrustedRoles extends Audit {
+
+  /**
+   *
+   */
+  public function audit(Sandbox $sandbox) {
+    // Get the machine names of the untrusted roles
+    $untrusted_roles = $sandbox->getParameter('untrusted_roles', ['anonymous', 'authenticated']);
+    
+    // Load all of Drupal's permissions so that we have access to the
+    // "restrict access" property.
+    $command = "echo json_encode(\\Drupal::service('user.permissions')->getPermissions());";
+    $command = "\"$command\"";
+    $all_permissions = $sandbox->drush(['format' => 'json'])->ev($command);
+
+    $rows = [];
+    foreach ($untrusted_roles as $role) {
+      $untrusted_permissions = [];
+      // Get all permissions assigned to the untrusted role.
+      $role_permissions = $sandbox->drush(['format' => 'json'])
+        ->roleList($role);
```

Comment:
This can be quite taxing since it requires multiple drush calls. Consider the anonymous function I used above as potentially a place to conduct this logic also so that multiple remote calls are not required.

Reply:
Yes! I was hoping you would have a better solution to minimize remote calls.